### PR TITLE
add hit the API

### PR DESCRIPTION
**Pull request by wolterbwambale**
**in this project, implemented the following:-**

1. I  create a new game with the name using the API.

2. I  saved the ID of my game that will be returned by API.

3. Implement the "Refresh" button that will get all scores for a game created by you from the API (receiving data from the API and parsing the JSON).
4. Implement the form "Submit" button that will save a score for a game created by you (sending data to the API).
5. I use arrow functions instead of the function keyword.
6. I use async and await JavaScript features to consume the API.
7.No styling is required.

### CODE REVIEW

File: src/display.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    scoreCell.textContent = score.score;
+
+    row.appendChild(nameCell);
+    row.appendChild(scoreCell);
+    tableBody.appendChild(row);
+  });
+
+  table.appendChild(tableBody);
+  resultsDiv.appendChild(table);
+};
```
Comment (abdulhamiid) [inline src/display.js]: Please kindly fix the error in the console 🤗🛠

image

File: src/form.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      const data = await response.json();
+      console.log('Score submitted:', data);
+      refreshScores();
+    } else {
+      console.error('Error:', response.status);
+    }
+  } catch (error) {
+    console.error('Error:', error);
+  }
+};
```
Comment (abdulhamiid) [inline src/form.js]: - In your app, the refresh button is not working 👀
- According to the project requirements, kindly Implement the "Refresh" button to get all scores from the API. 🛠
- When the Refresh button is click, it should hit the GET API endpoint causing the scores list to change.

Note:
- The "Submit" button will post user entries to the API and save them
- The "Refresh" button will fetch the scores and display them in the UI. 🤗

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @wolterbwambale 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! :running_man: :runner: 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Descriptions are very descriptive
- ✔️ Linters checks are passing
- ✔️ Use Git Flow

## Required Changes ♻️
_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 ✔️ 

## Hi @wolterbwambale 👋, 

Great job implementing the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**